manifests/k/KiCad/KiCad/7.0.0/KiCad.KiCad.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: KiCad.KiCad
+PackageVersion: 7.0.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+FileExtensions:
+- dcm
+- drl
+- kicad_lib
+- kicad_mod
+- kicad_pcb
+- kicad_wks
+- net
+- pro
+- sch
+ReleaseDate: 2023-02-13
+Installers:
+- Architecture: x64
+  InstallerUrl: https://s3.cern.ch/kicad-downloads/windows/stable/kicad-7.0.0-x86_64.exe
```

Comment:
```suggestion
  InstallerUrl: https://github.com/KiCad/kicad-source-mirror/releases/download/7.0.0/kicad-7.0.0-x86_64.exe
```

Reply:
Hi @kdp

Our [Submission expectations](https://learn.microsoft.com/en-us/windows/package-manager/package/repository#validation-process) doesn't really allow mirrors. The developer's GitHub (or other public repo) is very probably under the developer's control, making it a more preferred source.